### ISSUE TITLE
fix remaining non-indexed people

### ISSUE THREAD

**Issue by mhl** [mysociety/yournextmp-popit#317]
The following people are in PopIt, but can't be found by search:

* Found 0 people with ID 1419 http://yournextmp.popit.mysociety.org/api/v0.1/persons/1419 - search URL was: http://yournextmp.popit.mysociety.org/api/v0.1/search/persons?q=id%3A1419
* Found 0 people with ID 4621 http://yournextmp.popit.mysociety.org/api/v0.1/persons/4621 - search URL was: http://yournextmp.popit.mysociety.org/api/v0.1/search/persons?q=id%3A4621
* Found 0 people with ID 4620 http://yournextmp.popit.mysociety.org/api/v0.1/persons/4620 - search URL was: http://yournextmp.popit.mysociety.org/api/v0.1/search/persons?q=id%3A4620
* Found 0 people with ID 5222 http://yournextmp.popit.mysociety.org/api/v0.1/persons/5222 - search URL was: http://yournextmp.popit.mysociety.org/api/v0.1/search/persons?q=id%3A5222
* Found 0 people with ID 5394 http://yournextmp.popit.mysociety.org/api/v0.1/persons/5394 - search URL was: http://yournextmp.popit.mysociety.org/api/v0.1/search/persons?q=id%3A5394
* Found 0 people with ID 4971 http://yournextmp.popit.mysociety.org/api/v0.1/persons/4971 - search URL was: http://yournextmp.popit.mysociety.org/api/v0.1/search/persons?q=id%3A4971
* Found 0 people with ID 4613 http://yournextmp.popit.mysociety.org/api/v0.1/persons/4613 - search URL was: http://yournextmp.popit.mysociety.org/api/v0.1/search/persons?q=id%3A4613
* Found 0 people with ID 4222 http://yournextmp.popit.mysociety.org/api/v0.1/persons/4222 - search URL was: http://yournextmp.popit.mysociety.org/api/v0.1/search/persons?q=id%3A4222
* Found 0 people with ID 3245 http://yournextmp.popit.mysociety.org/api/v0.1/persons/3245 - search URL was: http://yournextmp.popit.mysociety.org/api/v0.1/search/persons?q=id%3A3245
* Found 0 people with ID 4614 http://yournextmp.popit.mysociety.org/api/v0.1/persons/4614 - search URL was: http://yournextmp.popit.mysociety.org/api/v0.1/search/persons?q=id%3A4614
* Found 0 people with ID 5157 http://yournextmp.popit.mysociety.org/api/v0.1/persons/5157 - search URL was: http://yournextmp.popit.mysociety.org/api/v0.1/search/persons?q=id%3A5157
* Found 0 people with ID 4554 http://yournextmp.popit.mysociety.org/api/v0.1/persons/4554 - search URL was: http://yournextmp.popit.mysociety.org/api/v0.1/search/persons?q=id%3A4554

**Comment by mhl** [comment]
An example of an update that indexing failed for would be:

```
[2015-04-13 08:47:44,092][DEBUG][action.bulk              ] [pluto] [popit_yournextmp_20150314153759][3] failed to execute bulk item (index) index {[popit_yournextmp_20150314153759][membership][5501f87255f04a60241b0c0c], source[{"end_date":"9999-12-31","html_url":"http://yournextmp.popit.mysociety.org/memberships/5501f87255f04a60241b0c0c","id":"5501f87255f04a60241b0c0c","organization_id":"party:52","person_id":"6752","start_date":"2010-05-07","url":"http://yournextmp.popit.mysociety.org/api/v0.1/memberships/5501f87255f04a60241b0c0c","images":[],"links":[],"contact_details":[],"person":{"birth_date":"0000-01-01","email":"info@sfcua.org.uk","gender":"male","honorific_prefix":null,"honorific_suffix":null,"html_url":"http://yournextmp.popit.mysociety.org/persons/6752","id":"6752","name":"Alex Stewart-Clark","party_memberships":{"2015":{"id":"party:52","name":"Conservative Party"}},"standing_in":{"2015":{"mapit_url":"http://mapit.mysociety.org/area/14432","name":"Glenrothes","post_id":"14432"}},"url":"http://yournextmp.popit.mysociety.org/api/v0.1/persons/6752","versions":[{"information_source":"Facebook + Google","timestamp":"2015-04-12T00:02:51.080727","username":"TimPatmore","data":{"facebook_page_url":"https://www.facebook.com/Alexstewartc?fref=ts","honorific_suffix":"","party_ppc_page_url":"https://www.conservatives.com/OurTeam/Prospective_Parliamentary_Candidates/Stewart_Clark_Alex.aspx","image":null,"facebook_personal_url":"https://www.facebook.com/alex.stewartclark?fref=ts","id":"6752","other_names":[],"proxy_image":null,"honorific_prefix":"","wikipedia_url":"","party_memberships":{"2015":{"name":"Conservative Party","id":"party:52"}},"email":"info@sfcua.org.uk","twitter_username":"StewartClark_","name":"Alex Stewart-Clark","gender":"male","identifiers":[],"linkedin_url":"","standing_in":{"2015":{"post_id":"14432","name":"Glenrothes","mapit_url":"http://mapit.mysociety.org/area/14432"}},"homepage_url":"http://www.stewart-clark.com/","birth_date":null},"version_id":"51c3d62e264306a0"},{"username":"RichCBury1","timestamp":"2015-03-12T20:34:58.780742","information_source":"http://www.stewart-clark.com/","data":{"twitter_username":"","name":"Alex Stewart-Clark","honorific_suffix":"","party_ppc_page_url":"","gender":"male","id":"6752","facebook_personal_url":"","standing_in":{"2015":{"post_id":"14432","name":"Glenrothes","mapit_url":"http://mapit.mysociety.org/area/14432"}},"honorific_prefix":"","linkedin_url":"","homepage_url":"http://www.stewart-clark.com/","wikipedia_url":"","party_memberships":{"2015":{"name":"Conservative Party","id":"party:52"}},"birth_date":null,"facebook_page_url":"","email":""},"version_id":"38504ca4233db3f6"}],"images":[],"links":[],"contact_details":[],"identifiers":[],"other_names":[],"birth_date_missing":true},"organization":{"classification":"Party","dissolution_date":"9999-12-31","founding_date":"1999-01-14","html_url":"http://yournextmp.popit.mysociety.org/organizations/party%3A52","id":"party:52","image":"http://yournextmp.popit.mysociety.org/organizations/party:52/image/54ce624f95590a10203202ff","name":"Conservative Party","proxy_image":"http://yournextmp.popit.mysociety.org/image-proxy/http%3A%2F%2Fyournextmp.popit.mysociety.org%2Forganizations%2Fparty%3A52%2Fimage%2F54ce624f95590a10203202ff","register":"Great Britain","slug":"conservative-party","url":"http://yournextmp.popit.mysociety.org/api/v0.1/organizations/party%3A52","images":[{"birth_date":"0000-01-01","created":"","death_date":"9999-12-31","dissolution_date":"9999-12-31","end_date":"9999-12-31","founding_date":"0000-01-01","id":"551d0e59e411884ace31c3f0","local_path":"54/ce/54ce624f95590a10203202ff","local_path_base":"54/ce/54ce624f95590a10203202ff","md5sum":"0c130ed06dfd2e113b3e08a6e8a1730f","mime_type":"image/gif","notes":"Emblem 1 Conservatives Emblem","proxy_url":"http://yournextmp.popit.mysociety.org/image-proxy/http%3A%2F%2Fyournextmp.popit.mysociety.org%2Forganizations%2Fparty%3A52%2Fimage%2F54ce624f95590a10203202ff","source":"The Electoral Commission","start_date":"0000-01-01","url":"http://yournextmp.popit.mysociety.org/organizations/party:52/image/54ce624f95590a10203202ff","birth_date_missing":true,"death_date_missing":true,"dissolution_date_missing":true,"end_date_missing":true,"founding_date_missing":true,"start_date_missing":true},{"birth_date":"0000-01-01","created":"","death_date":"9999-12-31","dissolution_date":"9999-12-31","end_date":"9999-12-31","founding_date":"0000-01-01","id":"551d0e59e411884ace31c3f1","local_path":"54/ce/54ce624f95590a1020320300","local_path_base":"54/ce/54ce624f95590a1020320300","md5sum":"b8c4ca17b07fbb88763b87ea93c69526","mime_type":"image/jpeg","notes":"Emblem 2 Scottish Emblem","proxy_url":"http://yournextmp.popit.mysociety.org/image-proxy/http%3A%2F%2Fyournextmp.popit.mysociety.org%2Forganizations%2Fparty%3A52%2Fimage%2F54ce624f95590a1020320300","source":"The Electoral Commission","start_date":"0000-01-01","url":"http://yournextmp.popit.mysociety.org/organizations/party:52/image/54ce624f95590a1020320300","birth_date_missing":true,"death_date_missing":true,"dissolution_date_missing":true,"end_date_missing":true,"founding_date_missing":true,"start_date_missing":true},{"birth_date":"0000-01-01","created":"","death_date":"9999-12-31","dissolution_date":"9999-12-31","end_date":"9999-12-31","founding_date":"0000-01-01","id":"551d0e59e411884ace31c3f2","local_path":"54/ce/54ce625095590a1020320301","local_path_base":"54/ce/54ce625095590a1020320301","md5sum":"fbbb72d2ef20fcc80f7071b13cb733f3","mime_type":"image/gif","notes":"Emblem 3 Welsh Emblem","proxy_url":"http://yournextmp.popit.mysociety.org/image-proxy/http%3A%2F%2Fyournextmp.popit.mysociety.org%2Forganizations%2Fparty%3A52%2Fimage%2F54ce625095590a1020320301","source":"The Electoral Commission","start_date":"0000-01-01","url":"http://yournextmp.popit.mysociety.org/organizations/party:52/image/54ce625095590a1020320301","birth_date_missing":true,"death_date_missing":true,"dissolution_date_missing":true,"end_date_missing":true,"founding_date_missing":true,"start_date_missing":true}],"posts":[],"links":[],"contact_details":[],"identifiers":[{"scheme":"electoral-commission","identifier":"PP52","id":"5520069cdcb7bda0778a9a1f"}],"other_names":[]}}]}
org.elasticsearch.index.mapper.MapperParsingException: failed to parse [organization.images.created]
        at org.elasticsearch.index.mapper.core.AbstractFieldMapper.parse(AbstractFieldMapper.java:396)
        at org.elasticsearch.index.mapper.object.ObjectMapper.serializeValue(ObjectMapper.java:599)
        at org.elasticsearch.index.mapper.object.ObjectMapper.parse(ObjectMapper.java:467)
        at org.elasticsearch.index.mapper.object.ObjectMapper.serializeObject(ObjectMapper.java:515)
        at org.elasticsearch.index.mapper.object.ObjectMapper.serializeArray(ObjectMapper.java:577)
        at org.elasticsearch.index.mapper.object.ObjectMapper.parse(ObjectMapper.java:459)
        at org.elasticsearch.index.mapper.object.ObjectMapper.serializeObject(ObjectMapper.java:515)
        at org.elasticsearch.index.mapper.object.ObjectMapper.parse(ObjectMapper.java:457)
        at org.elasticsearch.index.mapper.DocumentMapper.parse(DocumentMapper.java:507)
        at org.elasticsearch.index.mapper.DocumentMapper.parse(DocumentMapper.java:451)
        at org.elasticsearch.index.shard.service.InternalIndexShard.prepareIndex(InternalIndexShard.java:329)
        at org.elasticsearch.action.bulk.TransportShardBulkAction.shardIndexOperation(TransportShardBulkAction.java:396)
        at org.elasticsearch.action.bulk.TransportShardBulkAction.shardOperationOnPrimary(TransportShardBulkAction.java:155)
        at org.elasticsearch.action.support.replication.TransportShardReplicationOperationAction$AsyncShardOperationAction.performOnPrimary(TransportShardReplicationOperationAction.java:521)
        at org.elasticsearch.action.support.replication.TransportShardReplicationOperationAction$AsyncShardOperationAction$1.run(TransportShardReplicationOperationAction.java:419)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1146)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
        at java.lang.Thread.run(Thread.java:701)
Caused by: org.elasticsearch.index.mapper.MapperParsingException: failed to parse date field [], tried both date format [dateOptionalTime], and timestamp number with locale [null]
        at org.elasticsearch.index.mapper.core.DateFieldMapper.parseStringValue(DateFieldMapper.java:465)
        at org.elasticsearch.index.mapper.core.DateFieldMapper.innerParseCreateField(DateFieldMapper.java:413)
        at org.elasticsearch.index.mapper.core.NumberFieldMapper.parseCreateField(NumberFieldMapper.java:167)
        at org.elasticsearch.index.mapper.core.AbstractFieldMapper.parse(AbstractFieldMapper.java:385)
        ... 17 more
Caused by: java.lang.IllegalArgumentException: Invalid format: ""
        at org.elasticsearch.common.joda.time.format.DateTimeFormatter.parseMillis(DateTimeFormatter.java:752)
        at org.elasticsearch.index.mapper.core.DateFieldMapper.parseStringValue(DateFieldMapper.java:459)
        ... 20 more
```

This indexing error was as a result of PUTting the following record to persons/6752

```
{
    "birth_date": null,
    "contact_details": [],
    "email": "info@sfcua.org.uk",
    "gender": "male",
    "honorific_prefix": null,
    "honorific_suffix": null,
    "html_url": "http://yournextmp.popit.mysociety.org/persons/6752",
    "id": "6752",
    "identifiers": [],
    "images": [],
    "links": [],
    "memberships": [],
    "name": "Alex Stewart-Clark",
    "other_names": [],
    "party_memberships": {
        "2015": {
            "id": "party:52",
            "name": "Conservative Party"
        }
    },
    "standing_in": {
        "2015": {
            "mapit_url": "http://mapit.mysociety.org/area/14432",
            "name": "Glenrothes",
            "post_id": "14432"
        }
    },
    "url": "http://yournextmp.popit.mysociety.org/api/v0.1/persons/6752",
    "versions": [
        {
            "data": {
                "birth_date": null,
                "email": "info@sfcua.org.uk",
                "facebook_page_url": "https://www.facebook.com/Alexstewartc?fref=ts",
                "facebook_personal_url": "https://www.facebook.com/alex.stewartclark?fref=ts",
                "gender": "male",
                "homepage_url": "http://www.stewart-clark.com/",
                "honorific_prefix": "",
                "honorific_suffix": "",
                "id": "6752",
                "identifiers": [],
                "image": null,
                "linkedin_url": "",
                "name": "Alex Stewart-Clark",
                "other_names": [],
                "party_memberships": {
                    "2015": {
                        "id": "party:52",
                        "name": "Conservative Party"
                    }
                },
                "party_ppc_page_url": "https://www.conservatives.com/OurTeam/Prospective_Parliamentary_Candidates/Stewart_Clark_Alex.aspx",
                "proxy_image": null,
                "standing_in": {
                    "2015": {
                        "mapit_url": "http://mapit.mysociety.org/area/14432",
                        "name": "Glenrothes",
                        "post_id": "14432"
                    }
                },
                "twitter_username": "StewartClark_",
                "wikipedia_url": ""
            },
            "information_source": "Facebook + Google",
            "timestamp": "2015-04-12T00:02:51.080727",
            "username": "TimPatmore",
            "version_id": "51c3d62e264306a0"
        },
        {
            "data": {
                "birth_date": null,
                "email": "",
                "facebook_page_url": "",
                "facebook_personal_url": "",
                "gender": "male",
                "homepage_url": "http://www.stewart-clark.com/",
                "honorific_prefix": "",
                "honorific_suffix": "",
                "id": "6752",
                "linkedin_url": "",
                "name": "Alex Stewart-Clark",
                "party_memberships": {
                    "2015": {
                        "id": "party:52",
                        "name": "Conservative Party"
                    }
                },
                "party_ppc_page_url": "",
                "standing_in": {
                    "2015": {
                        "mapit_url": "http://mapit.mysociety.org/area/14432",
                        "name": "Glenrothes",
                        "post_id": "14432"
                    }
                },
                "twitter_username": "",
                "wikipedia_url": ""
            },
            "information_source": "http://www.stewart-clark.com/",
            "timestamp": "2015-03-12T20:34:58.780742",
            "username": "RichCBury1",
            "version_id": "38504ca4233db3f6"
        }
    ]
}
```

... which doesn't reference images, or an organization. However, the Elasticsearch error indicates that this is from indexing the membership of that person (which needs to be reindexed because it embeds person information) The error shows that the document PopIt is trying to index in Elasticsearch is actually:

```json
{
    "contact_details": [],
    "end_date": "9999-12-31",
    "html_url": "http://yournextmp.popit.mysociety.org/memberships/5501f87255f04a60241b0c0c",
    "id": "5501f87255f04a60241b0c0c",
    "images": [],
    "links": [],
    "organization": {
        "classification": "Party",
        "contact_details": [],
        "dissolution_date": "9999-12-31",
        "founding_date": "1999-01-14",
        "html_url": "http://yournextmp.popit.mysociety.org/organizations/party%3A52",
        "id": "party:52",
        "identifiers": [
            {
                "id": "5520069cdcb7bda0778a9a1f",
                "identifier": "PP52",
                "scheme": "electoral-commission"
            }
        ],
        "image": "http://yournextmp.popit.mysociety.org/organizations/party:52/image/54ce624f95590a10203202ff",
        "images": [
            {
                "birth_date": "0000-01-01",
                "birth_date_missing": true,
                "created": "",
                "death_date": "9999-12-31",
                "death_date_missing": true,
                "dissolution_date": "9999-12-31",
                "dissolution_date_missing": true,
                "end_date": "9999-12-31",
                "end_date_missing": true,
                "founding_date": "0000-01-01",
                "founding_date_missing": true,
                "id": "551d0e59e411884ace31c3f0",
                "local_path": "54/ce/54ce624f95590a10203202ff",
                "local_path_base": "54/ce/54ce624f95590a10203202ff",
                "md5sum": "0c130ed06dfd2e113b3e08a6e8a1730f",
                "mime_type": "image/gif",
                "notes": "Emblem 1 Conservatives Emblem",
                "proxy_url": "http://yournextmp.popit.mysociety.org/image-proxy/http%3A%2F%2Fyournextmp.popit.mysociety.org%2Forganizations%2Fparty%3A52%2Fimage%2F54ce624f95590a10203202ff",
                "source": "The Electoral Commission",
                "start_date": "0000-01-01",
                "start_date_missing": true,
                "url": "http://yournextmp.popit.mysociety.org/organizations/party:52/image/54ce624f95590a10203202ff"
            },
            {
                "birth_date": "0000-01-01",
                "birth_date_missing": true,
                "created": "",
                "death_date": "9999-12-31",
                "death_date_missing": true,
                "dissolution_date": "9999-12-31",
                "dissolution_date_missing": true,
                "end_date": "9999-12-31",
                "end_date_missing": true,
                "founding_date": "0000-01-01",
                "founding_date_missing": true,
                "id": "551d0e59e411884ace31c3f1",
                "local_path": "54/ce/54ce624f95590a1020320300",
                "local_path_base": "54/ce/54ce624f95590a1020320300",
                "md5sum": "b8c4ca17b07fbb88763b87ea93c69526",
                "mime_type": "image/jpeg",
                "notes": "Emblem 2 Scottish Emblem",
                "proxy_url": "http://yournextmp.popit.mysociety.org/image-proxy/http%3A%2F%2Fyournextmp.popit.mysociety.org%2Forganizations%2Fparty%3A52%2Fimage%2F54ce624f95590a1020320300",
                "source": "The Electoral Commission",
                "start_date": "0000-01-01",
                "start_date_missing": true,
                "url": "http://yournextmp.popit.mysociety.org/organizations/party:52/image/54ce624f95590a1020320300"
            },
            {
                "birth_date": "0000-01-01",
                "birth_date_missing": true,
                "created": "",
                "death_date": "9999-12-31",
                "death_date_missing": true,
                "dissolution_date": "9999-12-31",
                "dissolution_date_missing": true,
                "end_date": "9999-12-31",
                "end_date_missing": true,
                "founding_date": "0000-01-01",
                "founding_date_missing": true,
                "id": "551d0e59e411884ace31c3f2",
                "local_path": "54/ce/54ce625095590a1020320301",
                "local_path_base": "54/ce/54ce625095590a1020320301",
                "md5sum": "fbbb72d2ef20fcc80f7071b13cb733f3",
                "mime_type": "image/gif",
                "notes": "Emblem 3 Welsh Emblem",
                "proxy_url": "http://yournextmp.popit.mysociety.org/image-proxy/http%3A%2F%2Fyournextmp.popit.mysociety.org%2Forganizations%2Fparty%3A52%2Fimage%2F54ce625095590a1020320301",
                "source": "The Electoral Commission",
                "start_date": "0000-01-01",
                "start_date_missing": true,
                "url": "http://yournextmp.popit.mysociety.org/organizations/party:52/image/54ce625095590a1020320301"
            }
        ],
        "links": [],
        "name": "Conservative Party",
        "other_names": [],
        "posts": [],
        "proxy_image": "http://yournextmp.popit.mysociety.org/image-proxy/http%3A%2F%2Fyournextmp.popit.mysociety.org%2Forganizations%2Fparty%3A52%2Fimage%2F54ce624f95590a10203202ff",
        "register": "Great Britain",
        "slug": "conservative-party",
        "url": "http://yournextmp.popit.mysociety.org/api/v0.1/organizations/party%3A52"
    },
    "organization_id": "party:52",
    "person": {
        "birth_date": "0000-01-01",
        "birth_date_missing": true,
        "contact_details": [],
        "email": "info@sfcua.org.uk",
        "gender": "male",
        "honorific_prefix": null,
        "honorific_suffix": null,
        "html_url": "http://yournextmp.popit.mysociety.org/persons/6752",
        "id": "6752",
        "identifiers": [],
        "images": [],
        "links": [],
        "name": "Alex Stewart-Clark",
        "other_names": [],
        "party_memberships": {
            "2015": {
                "id": "party:52",
                "name": "Conservative Party"
            }
        },
        "standing_in": {
            "2015": {
                "mapit_url": "http://mapit.mysociety.org/area/14432",
                "name": "Glenrothes",
                "post_id": "14432"
            }
        },
        "url": "http://yournextmp.popit.mysociety.org/api/v0.1/persons/6752",
        "versions": [
            {
                "data": {
                    "birth_date": null,
                    "email": "info@sfcua.org.uk",
                    "facebook_page_url": "https://www.facebook.com/Alexstewartc?fref=ts",
                    "facebook_personal_url": "https://www.facebook.com/alex.stewartclark?fref=ts",
                    "gender": "male",
                    "homepage_url": "http://www.stewart-clark.com/",
                    "honorific_prefix": "",
                    "honorific_suffix": "",
                    "id": "6752",
                    "identifiers": [],
                    "image": null,
                    "linkedin_url": "",
                    "name": "Alex Stewart-Clark",
                    "other_names": [],
                    "party_memberships": {
                        "2015": {
                            "id": "party:52",
                            "name": "Conservative Party"
                        }
                    },
                    "party_ppc_page_url": "https://www.conservatives.com/OurTeam/Prospective_Parliamentary_Candidates/Stewart_Clark_Alex.aspx",
                    "proxy_image": null,
                    "standing_in": {
                        "2015": {
                            "mapit_url": "http://mapit.mysociety.org/area/14432",
                            "name": "Glenrothes",
                            "post_id": "14432"
                        }
                    },
                    "twitter_username": "StewartClark_",
                    "wikipedia_url": ""
                },
                "information_source": "Facebook + Google",
                "timestamp": "2015-04-12T00:02:51.080727",
                "username": "TimPatmore",
                "version_id": "51c3d62e264306a0"
            },
            {
                "data": {
                    "birth_date": null,
                    "email": "",
                    "facebook_page_url": "",
                    "facebook_personal_url": "",
                    "gender": "male",
                    "homepage_url": "http://www.stewart-clark.com/",
                    "honorific_prefix": "",
                    "honorific_suffix": "",
                    "id": "6752",
                    "linkedin_url": "",
                    "name": "Alex Stewart-Clark",
                    "party_memberships": {
                        "2015": {
                            "id": "party:52",
                            "name": "Conservative Party"
                        }
                    },
                    "party_ppc_page_url": "",
                    "standing_in": {
                        "2015": {
                            "mapit_url": "http://mapit.mysociety.org/area/14432",
                            "name": "Glenrothes",
                            "post_id": "14432"
                        }
                    },
                    "twitter_username": "",
                    "wikipedia_url": ""
                },
                "information_source": "http://www.stewart-clark.com/",
                "timestamp": "2015-03-12T20:34:58.780742",
                "username": "RichCBury1",
                "version_id": "38504ca4233db3f6"
            }
        ]
    },
    "person_id": "6752",
    "start_date": "2010-05-07",
    "url": "http://yournextmp.popit.mysociety.org/api/v0.1/memberships/5501f87255f04a60241b0c0c"
}
```

... so the real issue is that various organizations have images.created set to `""` instead of `null`, which causes the update of memberships to fail. If this indexing happens before the indexing of the person record, that would explain why these updates are failing.

Again, these errors would have been found a lot earlier if https://github.com/mysociety/popit-api/issues/97 were dealt with.

**Comment by mhl** [comment]
OK, so this seems to be fixed by ed19544cfe76fb0c64dc7 and running the fix image metadata script again so that it fixes organization images having `""` in `created` as well.
